generate/resources/couchbase-edge-server/README.md

```diff
@@ -1,16 +1,101 @@
-
 This README will guide you through running Couchbase Edge Server with Docker Containers.
 
-For configuration references or any additional information, please visit [Edge Server documentation site](https://docs.couchbase.com/edge-server/current/index.html).
+[Edge Server](#) is a lightweight self contained alternative to Sync Gateway that securely manages the synchronization of data between [Couchbase Lite](https://www.couchbase.com/products/lite), other Edge Server deployments and [Sync Gateway](https://www.couchbase.com/products/sync-gateway) designed to run in environments where a full cloud deployment is not optimal.
+
+For configuration references, or additional information about anything described below, visit the [Edge Server documentation site](#).
+
+For additional questions and feedback, please visit the [Couchbase Forums](https://forums.couchbase.com/c/mobile/edge-server).
+
+# QuickStart with Edge Server and Docker
+
+## Running Edge Server with Docker
 
-# Running Edge Server with Docker
+```
+$ docker run -d --name edge-server -p 59840:59840 couchbase/edge-server
+```
+
+At this point you should be able to send a HTTP request to the Edge Server on port `59840` using curl:
 
-1. Start Edge Server container with the default configuration file
 ```
-$ docker run --name edge-server couchbase/edge-server
+$ curl http://localhost:59840
+{"couchdb":"Welcome","vendor":{"name":"CouchbaseEdgeServer","version":"0.0.0 ()"},"version":"CouchbaseEdgeServer/0.0.0 () CouchbaseLiteCore/1.0.0 (21)"}
 ```
 
-2. Start Edge Server container with an external configuration file
+This setup is minimal and launches with an empty database, though any changes will be persisted.  Some key container paths are noted in the following diagram:
+
+![Default Docker Layout](./diagrams/docker-nomount.png)
+
+Where the bin and lib directories contain the necessary executable data to run the server, `/etc/config.json` is the configuration file that was used at startup, and `/var/databases` stores the resulting database specified by the configuration.
```

Comment:
I thought the paths are /opt/couchbase-edge-server/etc/config.json and /opt/couchbase-edge-server/var/databases

Reply:
They are.  I thought it was understood that I was talking in that context (since the diagram shows that).